packages/flutter/lib/src/material/icon_button.dart

```diff
@@ -301,10 +322,117 @@ class IconButton extends StatelessWidget {
   /// and `Theme.of(context).visualDensity` otherwise.
   final BoxConstraints? constraints;
 
+  /// {@macro flutter.material.ButtonStyleButton.style}
+  ///
+  /// [style] is only used for Material 3 IconButton. If [ThemeData.useMaterial3]
```

Comment:
nit: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#use-correct-grammar